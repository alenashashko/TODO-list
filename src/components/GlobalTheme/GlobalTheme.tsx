import { FC } from "react";
import { createMuiTheme, ThemeProvider, CssBaseline } from "@material-ui/core";

const fullHeight = {
  minHeight: "100%",
  height: "100%",
};

const theme = createMuiTheme({
  overrides: {
    MuiCssBaseline: {
      "@global": {
        html: fullHeight,
        body: fullHeight,
        "#root": fullHeight,
      },
    },
  },
  spacing: 4,
});

export const GlobalTheme: FC = ({ children }) => {
  return (
    <ThemeProvider theme={theme}>
      <CssBaseline />
      {children}
    </ThemeProvider>
  );
};
