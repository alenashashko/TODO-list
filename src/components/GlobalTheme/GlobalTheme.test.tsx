import { render, cleanup } from "@testing-library/react";
import { GlobalTheme } from "src/components/GlobalTheme";

afterEach(cleanup);

it("renders global theme", () => {
  const { asFragment } = render(<GlobalTheme />);

  expect(asFragment()).toMatchSnapshot();
});
