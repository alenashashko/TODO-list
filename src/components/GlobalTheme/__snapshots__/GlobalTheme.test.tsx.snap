// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders global theme 1`] = `<DocumentFragment />`;
